metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlHandler.java

```diff
@@ -70,6 +70,19 @@ public void setNamespace(final String namespace) {
         this.namespace = namespace;
     }
 
+    /**
+     * Sets whether to ignore the namespace.
+     *
+     * <strong>Default value: false</strong>
+     *
+     * @param ignoreNamespace true if the namespace should be ignored
+     */
+    public void setIgnoreNamespace(final boolean ignoreNamespace) {
+        if (ignoreNamespace) {
+            this.namespace = null;
```

Comment:
I don't think this is sufficient:

1. It becomes order-dependent; if one sets `ignoreNamespace` first and then `namespace`, the namespace check is still performed.
2. One can't reenable the namespace check without also setting the namespace to its previous value.

You would need to introduce a dedicated attribute for this.

Reply:
Fixed in [57f6a71](https://github.com/metafacture/metafacture-core/pull/575/commits/57f6a719a05265f485ac4b7bc57634d12bbbb02c).